.nycrc.json

```diff
@@ -1,17 +1,22 @@
 {
   "reporter": ["lcov"],
```

Comment:
what do folks think about adding a `text` and/or `text-summary` reporter for pretty printing of coverage in the terminal ? (unless that is already available and I just haven't found it yet 😄 )

Reply:
Sure! The only thing which makes our setup a little tricky is that there are four separate tests runs which all contribute to coverage. That's why we use `clean: true`. I'm not sure how that might interact with the terminal reporters.

Want to push some more commits to this branch?

Reply:
Ah yes I can do that now can't I!